src/services/event.ts

```diff
@@ -444,3 +445,112 @@ export async function findAllEvents(): Promise<EventShape[]> {
     raw: true,
   });
 }
+
+const splitPipe = (str: string) => str.split('\n').map((s) => s.trim()).filter(Boolean);
+
+const mappings: Record<string, string> = {
+  Audience: 'audience',
+  Creator: 'creator',
+  'Edit Title': 'eventName',
+  'Event Duration/#NC Days of Support': 'eventDuration',
+  'Event ID': 'eventId',
+  'Overall Vision/Goal for the PD Event': 'vision',
+  'Reason for Activity': 'reasons',
+  'Target Population(s)': 'targetPopulations',
+  'Event Organizer - Type of Event': 'eventOrganizer',
+};
+
+const toSplit = ['targetPopulations', 'reasons'];
+
+const replacements: Record<string, string> = {
```

Comment:
Do we want to add some tests for the auto replacement stuff?

Reply:
This meant, add a row to the unit testing file for this use case.

Reply:
The tests account for this now